.circleci/config.yml

```diff
@@ -101,6 +105,7 @@ workflows:
   build-test-and-publish:
     jobs:
       - checkout:
+          context: production
```

Comment:
What are contexts for BTW?

Reply:
Circle CI has this feature to define contexts for securing and sharing environment variables. These contexts are set on the organization and can be used across projects. So define once, use everywhere. Would be very helpful if the keys need to be rotated.

In our case I have defined contexts for each of our target environments.

more info: https://circleci.com/docs/2.0/contexts/